src/App.js

```diff
@@ -1,10 +1,18 @@
 import React from 'react';
 import './App.css';
 import { Home, About, Contact, Navigation } from './components';
+import {
+  BrowserRouter as Router,
+  Route
+} from 'react-router-dom'
+
 
 const App = () => (
   <div>
     <Navigation />
+    <Route path='/' exact component={Home}/>
+    <Route path='/about' exact component={About}/>
+    <Route path='/contact' exact component={Contact}/>  
```

Comment:
this won't break anything, but understand that only Home needs the `exact` prop with these current routes, since `/about` and `/contact` could never simultaneously match with any other route